include/nbl/video/SPhysicalDeviceFeatures.h

```diff
@@ -0,0 +1,508 @@
+
+    /* FragmentDensityMap2FeaturesEXT *//* VK_EXT_fragment_density_map2 */
+    //VkBool32           fragmentDensityMapDeferred;
+
+    /* FragmentShaderInterlockFeaturesEXT *//* VK_EXT_fragment_shader_interlock */
+    bool fragmentShaderSampleInterlock = false;
+    bool fragmentShaderPixelInterlock = false;
+    bool fragmentShaderShadingRateInterlock = false;
+
+    /* GlobalPriorityQueryFeaturesEXT *//* VK_EXT_global_priority */
+    /* GlobalPriorityQueryFeaturesKHR *//* VK_KHR_global_priority */
+    //VkQueueGlobalPriorityKHR    globalPriority;
+
+    /* GraphicsPipelineLibraryFeaturesEXT *//* VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibrary;
+
+    /* HostQueryResetFeatures *//* VK_EXT_host_query_reset *//* MOVED TO Vulkan 1.2 Core */
+    
+    /* Image2DViewOf3DFeaturesEXT *//* VK_EXT_image_2d_view_of_3d */
+    //VkBool32           image2DViewOf3D;
+    //VkBool32           sampler2DViewOf3D;
+
+    /* ImageRobustnessFeaturesEXT *//* VK_EXT_image_robustness *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ImageViewMinLodFeaturesEXT *//* VK_EXT_image_view_min_lod */
+    //VkBool32           minLod;
+
+    /* IndexTypeUint8FeaturesEXT *//* VK_EXT_index_type_uint8 */
+    //VkBool32           indexTypeUint8;
+    
+    /* InlineUniformBlockFeaturesEXT *//* VK_EXT_inline_uniform_block *//* MOVED TO Vulkan 1.3 Core */
+
+    /* LineRasterizationFeaturesEXT *//* VK_EXT_line_rasterization */
+    //VkBool32           rectangularLines;
+    //VkBool32           bresenhamLines;
+    //VkBool32           smoothLines;
+    //VkBool32           stippledRectangularLines;
+    //VkBool32           stippledBresenhamLines;
+    //VkBool32           stippledSmoothLines;
+
+    /* MemoryPriorityFeaturesEXT *//* VK_EXT_memory_priority */
+    //VkBool32           memoryPriority;
+
+    /* MultiDrawFeaturesEXT *//* VK_EXT_multi_draw */
+    //VkBool32           multiDraw;
+
+    /* PageableDeviceLocalMemoryFeaturesEXT *//* VK_EXT_pageable_device_local_memory */
+    //VkBool32           pageableDeviceLocalMemory;
+
+    /* PipelineCreationCacheControlFeaturesEXT *//* VK_EXT_pipeline_creation_cache_control *//* MOVED TO Vulkan 1.3 Core */
+
+    /* PrimitivesGeneratedQueryFeaturesEXT *//* VK_EXT_primitives_generated_query */
+    //VkBool32           primitivesGeneratedQuery;
+    //VkBool32           primitivesGeneratedQueryWithRasterizerDiscard;
+    //VkBool32           primitivesGeneratedQueryWithNonZeroStreams;
+
+    /* PrimitiveTopologyListRestartFeaturesEXT *//* VK_EXT_primitive_topology_list_restart */
+    //VkBool32           primitiveTopologyListRestart;
+    //VkBool32           primitiveTopologyPatchListRestart;
+
+    /* PrivateDataFeatures *//* VK_EXT_private_data *//* MOVED TO Vulkan 1.3 Core */
+
+    /* ProvokingVertexFeaturesEXT *//* VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexLast;
+    //VkBool32           transformFeedbackPreservesProvokingVertex;
```

Comment:
`provokingVertexLast` will not expose ( we always use First Vertex Vulkan-like convention)

anything to do with XForm-feedback we don't expose